base/src/androidLibMain/kotlin/ApplicationHolder.kt

```diff
@@ -0,0 +1,46 @@
+package com.splendo.kaluga.base
+
+import android.app.Application
+import android.content.Context
+
+/*
+
+Copyright 2019 Splendo Consulting B.V. The Netherlands
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+*/
+
+class ApplicationHolder {
```

Comment:
@arnoid can you add tests for this class?

https://github.com/splendo/kaluga/issues/90